RN 44601 (#5850)
SkylineCommunications/dataminer-docs: develop/devguide/Visio/linking_shapes/Linking_a_shape_to_a_matrix_parameter.md

# Linking a shape to a matrix parameter

By linking shapes in a visual overview to a matrix parameter of an element, you can create an interactive graphical representation of that matrix.

Each shape represents an input or an output of the matrix, while connections between shapes represent the active crosspoints. These connections between inputs and outputs are drawn automatically.

> Depending on the configuration, users can alter connections directly from the visual overview using the right-click menu. For example, from DataMiner 10.5.0 [CU12]/10.6.0/10.6.3 onwards<!--RN 44601-->, you can alter connections for table-based matrices. In the context menu, the outputs and inputs of these matrices are shown in the same order as the rows in the table.

Reviewer: This might need to be clarified a bit more. Is altering connections directly from Visual Overview possible for all parameter-based matrices, regardless of the DataMiner version? If yes, I'd rephrase this as follows:
```suggestion
Users can alter connections directly from the visual overview using the right-click menu. However, note that for table-based matrices this is only possible from DataMiner 10.5.0 [CU12]/10.6.0/10.6.3 onwards<!--RN 44601-->. In the context menu, the outputs and inputs of these matrices are shown in the same order as the rows in the table.
```
